src/components/bottom-bar.tsx

```diff
@@ -72,8 +76,13 @@ export class BottomBar extends BaseComponent<IProps, IState> {
         <div className={css.mainContainer}>
           {
             config.seasonButton &&
-            <div className={css.widgetGroup}>
-              <SeasonButton />
+            <div className={`${css.widgetGroup} hoverable ${seasonButtonHoveredClass}`}>
+              <SeasonButton
+                onMenuOpen={() => this.setState({ isSeasonMenuOpen: true })}
+                onMenuClose={() => {
+                  // delay to avoid flash between closing menu and :hover taking over
+                  setTimeout(() => this.setState({ isSeasonMenuOpen: false }), 500);
+                }} />
```

Comment:
Nice attention to details :wink: I was actually curious how this would look without this special handling, so I just set `seasonButtonHoveredClass` to "", and I didn't notice a difference. Open select kept the container hovered somehow. But I bet there're some edge cases I haven't seen.